fastapi/security/oauth2.py

```diff
@@ -595,6 +595,114 @@ async def __call__(self, request: Request) -> Optional[str]:
         return param
 
 
+class OAuth2ClientCredentials(OAuth2):
+    """
+    OAuth2 flow for authentication using a bearer token obtained with an OAuth2 client
+    credentials flow. An instance of it would be used as a dependency.
+    """
+
+    def __init__(
+        self,
+        tokenUrl: Annotated[
+            str,
+            Doc(
+                """
+                The URL to obtain the OAuth2 token.
+                """
+            ),
+        ],
+        refreshUrl: Annotated[
```

Comment:
Client Credentials do not have the concept of refresh (the refresh is done by obtaining a new token) so this parameter should be absent

Reply:
Thanks for spotting this I've removed this now. I originally included it as the `OAuthFlowClientCredentials` model includes this field (inherited from the OAuthFlow model) not sure if this should be changed too.

https://github.com/tiangolo/fastapi/blob/master/fastapi/openapi/models.py#L365-L379